web.beans/licenseinfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/web/beans/resources/delegate.png</file>
+        <file>src/org/netbeans/modules/web/beans/resources/injection_point.png</file>
+        <file>src/org/netbeans/modules/web/beans/resources/event.png</file>
+        <file>src/org/netbeans/modules/web/beans/resources/observer.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/beans/resources/Interceptor.template</file>
+        <file>src/org/netbeans/modules/web/beans/resources/Qualifier.template</file>
+        <file>src/org/netbeans/modules/web/beans/resources/Scope.template</file>
+        <file>src/org/netbeans/modules/web/beans/resources/Stereotype.template </file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="TEMPLATE_MINIMAL_IP"/>
+    </fileset>
+    <fileset>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/alternatives-beans-orig.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/alternatives-beans.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/beans-orig.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/beans.xml</file>
+        <file>test/unit/src/org/netbeans/modules/web/beans/xdm/model/empty-beans.xml</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="TEMPLATE_MINIMAL_IP"/>        
```

Comment:
I'm not sure whether this is correct.
@matthiasblaesing Is this OK?

Reply:
That's a problem, because this module depends on several other ones. I will remove that part form the lisenceinfo file.

Reply:
I do this:
- build the full netbeans cluster - in the root of the checkout run `ant -Dcluster.config=full` this takes about 15 minutes on my notebook
- open the module then in the IDE and check the project properties, make sure a JDK 8 is select in Libraries -> Java Platform
- The you can run the unittests via "Test" from the projects context menu

Reply:
It took 50 minutes + 10 minutes with test (old notebook). Have two test methods failed before and after. I will let someone with more experience verify the test after merge.

Reply:
Thanks for your work! I also verified that the two test methods failed. Please write about that in the confluence page. If you cannot edit it, please ask @geertjanw .